src/locales/en.json

```diff
@@ -360,15 +360,15 @@
       },
       "useCase_3": {
         "title": "API access to different types of points of interest",
-        "description": "The Panoramax API allows any site or application to easily integrate a photo of a place or point of interest as an illustration.\n\nThis is what example <a href='https://cartes.app/' target='_blank' rel='noopener'>cartes.app</a>",
+        "description": "The Panoramax API allows any site or application to easily integrate a photo of a place or point of interest as an illustration.\n\nThis is an example: <a href='https://cartes.app/' target='_blank' rel='noopener'>cartes.app</a>",
         "image": {
           "url": "api-white.svg",
           "alt": "API icon"
         }
       },
       "useCase_4": {
         "title": "Where we're going, we don't need a road",
-        "description": "Why limit yourself to the road domain?\nPhoto-mapping waterways, landscapes, the interior of forests, complementing day photos with night photos allows new and innovative uses .\nThe study of the banks, the underside of bridges, the state of public lighting... all this can also be done with Panoramax!",
+        "description": "Why limit yourself to the road?\nPhoto-mapping waterways, landscapes, forest paths, complementing daytime photos with nightime photos allows new and innovative uses .\nThe study of the riverbanks, the undersides of bridges, the state of public lighting... all of this can also be done with Panoramax!",
```

Comment:
```suggestion
        "description": "Why limit yourself to the road?\nPhoto-mapping waterways, landscapes, forest paths, complementing daytime photos with nighttime photos allows new and innovative uses .\nThe study of the riverbanks, the undersides of bridges, the state of public lighting... all of this can also be done with Panoramax!",
```